### PR TITLE
get rid of the `screens` concept

### PR DESCRIPTION
**Pull request by rfomin**
We are using SDL2 with only one "screen". Switch to Chocolate Doom like `I_VideoBuffer` system.

### CODE REVIEW

File: src/hu_stuff.c [hunk 1]
```diff
@@ -1476,8 +1476,8 @@ void WI_DrawTimeWidget(void)
 void HU_Erase(void)
 {
   // [FG] TODO optimize!
-  if (!automapactive && viewwindowx)
-    R_DrawViewBorder();
+  // if (!automapactive && viewwindowx)
+  //   R_DrawViewBorder();
```
Comment (rfomin) [inline src/hu_stuff.c:1480]: @fabiangreffrath Why do we need this? Seems to work without it.

Comment (fabiangreffrath) [reply src/hu_stuff.c:1480]: The boom HUD draws on the border and this makes sure to clean it up.

Comment (rfomin) [reply src/hu_stuff.c:1480]: Oh, there were `HUlib_erase` functions. I guess it's not worth optimizing.

**Review by fabiangreffrath — APPROVED** [review]
Very nice clean up, thank you!

**Pull request merged**